app-backend/common/src/main/scala/datamodel/ProjectLayer.scala

```diff
@@ -55,7 +63,9 @@ object ProjectLayer extends LazyLogging {
                           smartLayerId: Option[UUID],
                           rangeStart: Option[Timestamp],
                           rangeEnd: Option[Timestamp],
-                          geometry: Option[Projected[Geometry]]) {
+                          geometry: Option[Projected[Geometry]],
+                          isSingleBand: Boolean,
```

Comment:
So this seems to break v2 frontend for creating new layers from within a project, I propose two solutions (there might be better solutions though):
1. instead of doing the `@JsonCodec` semi-automatic generic derivation, we write our own custom `circe` decoder (similar to what we do to [`Project.Create`](https://github.com/raster-foundry/raster-foundry/blob/develop/app-backend/common/src/main/scala/datamodel/Project.scala#L159-L160))
2. update frontend logic and UI to add a checkbox in the layer creation modal

I personally lean towards the first one, since a single band layer should be triggered through changing color mode/correction.

Reply:
Decoders created in fd0f858 and 064ad0f